src/style.css

```
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  margin-inline: 22px;
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  display: flex;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
```

and here need to combine these 5 classes in on rule
```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2) {
  background-image: url(./images/star-active.svg);
}
```